src/components/ListCourses.js

```diff
@@ -12,15 +12,15 @@ function ListCourses(props) {
         className="group-list-item"
       >
         <button
-          className="select-list"
+          className={"select-list " + (props.selectedCourse === course ? "selected-color" : "")}
```

Comment:
Better to use a template string here. 

`className={`select-list ${props.selectedCourse === course ? 'selected-color' : ''`}}`